docker_test_tools/wiremock.py

```diff
@@ -105,13 +116,21 @@ def get_request_journal(self):
         response_body = json.loads(response.text)
         return response_body["requests"]
 
-    def get_matching_requests(self, inner_url):
+    def get_matching_requests(self, inner_url=None, stub_id=None):
         """Get all wiremock service requests of the given type (by inner URL) from  the journal.
 
         :param inner_url: The inner URL with which to filter journal requests by matching.
+        :param stub_id: The matched stub id with which to filter journal requests by matching.
         """
-        return [request for request in self.get_request_journal() if
-                request["request"]["url"] == inner_url]
+        matching_requests = []
+        for request in self.get_request_journal():
+            if inner_url is not None and request["request"]["url"] != inner_url:
```

Comment:
`if inner_url is not None` -> `if inner_url`

Reply:
I prefer to be explicit. 
This avoids ambiguity in case a valid value can also possibly be evaluated as false (e.g inner_url='')